site-manage/datemanager/tool/src/main/java/org/sakaiproject/datemanager/tool/MainController.java

```diff
@@ -47,10 +52,29 @@
 @Controller
 public class MainController {
 
-	@Inject private DateManagerService dateManagerService;
-
-	@RequestMapping(value = {"/", "/index"}, method = RequestMethod.GET)
-	public String showIndex(@RequestParam(required=false) String code, Model model) {
+    @Inject private DateManagerService dateManagerService;
+	
+    @Autowired
+    private SessionManager sessionManager;
+    
+    @Autowired
+    private PreferencesService preferencesService;
+
+    @RequestMapping(value = {"/", "/index"}, method = RequestMethod.GET)
+    public String showIndex(@RequestParam(required=false) String code, Model model, HttpServletRequest request, HttpServletResponse response) {
+		
+        Locale.setDefault(Locale.US);
```

Comment:
This is an anti pattern. You should never need to do this as Locale.getDefault picks up the settings from the JVM.

Reply:
or even more so get it from the PreferencesService